### PR TITLE
[WIP] Do not use hive.default as default catalog in product tests

### PR DESCRIPTION
**Pull request by wendigo**
This will probably fail

Fixes https://github.com/trinodb/trino/issues/18126

### CODE REVIEW

File: testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-tls.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ databases:
     http_port: 8080
     https_port: ${databases.presto.port}
     server_address: https://${databases.presto.host}:${databases.presto.port}
-    jdbc_url: "jdbc:trino://${databases.presto.host}:${databases.presto.port}/hive/${databases.hive.schema}?\
+    jdbc_url: "jdbc:trino://${databases.presto.host}:${databases.presto.port}?\
```
Comment (findepi) [inline testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/tempto/tempto-configuration-for-docker-tls.yaml:8]:
```suggestion
    jdbc_url: "jdbc:trino://${databases.presto.host}:${databases.presto.port}/?\
```

?

File: testing/trino-product-tests/src/main/resources/tempto-configuration.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ databases:
     port: 8080
     server_address: http://${databases.presto.host}:${databases.presto.port}
     jdbc_driver_class: io.trino.jdbc.TrinoDriver
-    jdbc_url: jdbc:trino://${databases.presto.host}:${databases.presto.port}/hive/${databases.hive.schema}
+    jdbc_url: jdbc:trino://${databases.presto.host}:${databases.presto.port}
```
Comment (findepi) [inline testing/trino-product-tests/src/main/resources/tempto-configuration.yaml:31]: nit?

```suggestion
    jdbc_url: jdbc:trino://${databases.presto.host}:${databases.presto.port}/
```

**Comment by ebyhr** [comment]
It would be better to run all tests with secret before merging this PR.

**Comment by findepi** [comment]
good point @ebyhr . @wendigo can you perhaps close this PR and re-raise from within the repo?

**Pull request closed without merging**